scripts/homepage.js

```diff
@@ -230,19 +238,21 @@ $(function () {
         var translate_download = $('#translate-download').text();
         var translate_purchase = $('#translate-purchase').text();
 
-        if (
-            $('input#amount-custom').hasClass('checked') &&
-            payment_amount < payment_minimum &&
-            $('#amount-custom').val() != ''
-        ) {
+        if ($('#amounts').children().length <= 1) {
```

Comment:
We never did this before because the default selection once paid is 0, which fails the `>= payment_minimum` check.